lib/mzgl/gl/DefaultFont.inc.h

```diff
@@ -7,8 +7,8 @@
 //
 // generated from this command: xxd -i ProggySquare.ttf
 
-vector<unsigned char> getDefaultFontData() {
-	vector<unsigned char> defaultFontData = {
+std::vector<unsigned char> getDefaultFontData() {
+	std::vector<unsigned char> defaultFontData = {
```

Comment:
I agree but once its an array how do I pass to a method that only takes a vector? - (I think this only gets used if you turn on debug text in koala and it will happen once)